### PR TITLE
Update for Zemismart Chain Roller Shades Driver M515EGB (TS0601)

### PR DESCRIPTION
**Pull request by Smanar**
This device have 2 model, the wired one, router, and the battery one, end device (both using same model ID, manufacture name)
This code code permit the inclusion for the battery powered one.

### CODE REVIEW

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -2219,8 +2219,9 @@ void DeRestPluginPrivate::addLightNode(const deCONZ::Node *node)
             {
                 hasServerOnOff = false;
             }
-            //to test
-            if (lightNode.manufacturer() == QLatin1String("_TZE200_wmcdj3aq"))
+            //Covering
+            if ((lightNode.manufacturer() == QLatin1String("_TZE200_wmcdj3aq")) ||
+                (lightNode.manufacturer() == QLatin1String("_TZE200_xuzcvlku")) ) // Zemismart Chain Roller Shades Driver M515EGB (TS0601)
```
Comment (manup) [inline de_web_plugin.cpp:2224]: White space, parentheses ;)

otherwise fine to merge

**Review by manup — CHANGES_REQUESTED** [review]

**Comment by manup** [comment]
Thanks, but git is still complaining conflicts, can you please sync the PR with master.

**Comment by Smanar** [comment]
Done

**Comment by jscheurwater** [comment]
@Smanar I'm pretty new here and have the DeconZ with Phoscon setup on my Raspberry PI. I just wanted to pair the Zemismart M515EGB (battery powered) Zigbee device to the Phoscon app.

Like I just read here, you made some kind of modifications to the DeconZ rest API to make it possible to include this device, right?

Can you please tell me how I can use your code modification on my Raspberry PI Deconz Phoscon setup? 
Is this an update I have to do manually or will this modification be in the next update of Deconz?

Thanks in advance!

**Comment by Smanar** [comment]
It seem you are not alone with the problem.
We are making tries here > https://github.com/dresden-elektronik/deconz-rest-plugin/pull/4223#issuecomment-782835076

**Review by manup — APPROVED** [review]

**Pull request merged**